code/modules/organs/organ_alien.dm

```diff
@@ -171,7 +171,8 @@
 /datum/internal_organ/stack/process()
 	if(owner && owner.stat != 2 && !is_broken())
 		backup_time = world.time
-		if(owner.mind) backup = owner.mind
+		if(owner.mind) 
```

Comment:
y no `?.` ?

Reply:
its checking for owner.mind, not mind.something